src/view/viewTabs.component.test.tsx

```diff
@@ -20,13 +20,7 @@ describe('View Tabs', () => {
     expect(viewTabs.getByRole('tab', { selected: true })).toHaveTextContent(
       'Catalogue'
     );
-    const catalogueHomeButton = await screen.findByRole('button', {
-      name: 'navigate to catalogue home',
-    });
-    expect(catalogueHomeButton).toBeInTheDocument();
-    await waitFor(() => {
-      expect(screen.getByText('Beam Characterization')).toBeInTheDocument();
-    });
```

Comment:
Why was this removed? It ensures the routing renders the appropriate component. Same with the other removals below.